.travis.yml

```diff
@@ -38,17 +38,17 @@ matrix:
     # This environment tests the newest supported Anaconda release (4.4.0)
     # It also runs tests requiring Pandas.
     - env: DISTRIB="conda" PYTHON_VERSION="3.6.1" INSTALL_MKL="true"
-           NUMPY_VERSION="1.12.1" SCIPY_VERSION="0.19.0" PANDAS_VERSION="0.20.1"
+           NUMPY_VERSION="1.13" SCIPY_VERSION="0.19.0" PANDAS_VERSION="0.20.1"
```

Comment:
bumping pandas to 0.20.2 should do it, according to [this](https://anaconda.org/anaconda/pandas/files). (don't forget the commented out env below)